tests/gtest/datahandle_attribute_test.cpp

```diff
@@ -233,4 +277,59 @@ TEST_F(DatahandleAttributeTest, Attribute_Different_Size_Double) {
     delete subvolume;
 }
 
+TEST_F(DatahandleAttributeTest, Attribute_Double_Data_Outside_Intersection_Horizontal_Plane) {
+    DoubleDataHandle& datahandle = double_datahandle;
+    int ilines_before = 3;
+    int ilines_after = 1;
+    int xlines_before = 1;
+    int xlines_after = 2;
+
+    Grid grid = get_grid(datahandle, -ilines_before, -xlines_before);
+
+    const MetadataHandle* metadata = &(datahandle.get_metadata());
+
+    std::size_t nrows = metadata->iline().nsamples() + ilines_before + ilines_after;
+    std::size_t ncols = metadata->xline().nsamples() + xlines_before + xlines_after;
+    static std::vector<float> top_surface_data(nrows * ncols, 28.0f);
+    static std::vector<float> pri_surface_data(nrows * ncols, 36.0f);
+    static std::vector<float> bot_surface_data(nrows * ncols, 52.0f);
+    RegularSurface pri_surface = RegularSurface(pri_surface_data.data(), nrows, ncols, grid, fill);
+    RegularSurface top_surface = RegularSurface(top_surface_data.data(), nrows, ncols, grid, fill);
+    RegularSurface bot_surface = RegularSurface(bot_surface_data.data(), nrows, ncols, grid, fill);
+    SurfaceBoundedSubVolume* subvolume = make_subvolume(datahandle.get_metadata(), pri_surface, top_surface, bot_surface);
+
+    cppapi::fetch_subvolume(datahandle, *subvolume, NEAREST, 0, nrows * ncols);
+
+    int expected_low[3] = {4, 4, 4};
+    int expected_high[3] = {8, 8, 15};
+
+    int requested_low[3] = {expected_low[0] - ilines_before, expected_low[1] - xlines_before, expected_low[2]};
+    int requested_high[3] = {expected_high[0] + ilines_after, expected_high[1] + xlines_after, expected_high[2]};
+
+    check_attribute(*subvolume, requested_low, requested_high, expected_low, expected_high, 2);
+
+    delete subvolume;
+}
+
+TEST_F(DatahandleAttributeTest, Attribute_Double_Data_Outside_Intersection_Vertical_Plane) {
```

Comment:
I think it would be useful with a short explanation of what actually goes on in this test. I assume one of the surfaces within the bounds of one of the intersecting cubes, but outside the bounds of the intersection?

Reply:
Still missing a comment here

Reply:
I added "lowest sample belonging to intersection is 20" that confirms your assumption.

Given that point of the test is to check that when you request data outside of the intersection in vertical plane, you get an error, I am not sure what more besides this comment and test name to add here.

Reply:
I don't see that comment anywhere

Reply:
https://github.com/equinor/vds-slice/blob/73c191341ce35a7c36a98eb7e605f6f52be97bce/tests/gtest/datahandle_attribute_test.cpp#L332

Reply:
Weird, I don't see it in any of the fixups, but I do see it in "files changed"

Reply:
First fixup, second change from the bottom.